recipes/dpath/meta.yaml

```diff
@@ -0,0 +1,107 @@
+{% set name = "dpath" %}
+{% set version = "1.4.0" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "89fbec229d941e622ba4e9578a79e05f8e0b9efdfe40ffbf4e670a129ec54d6d" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
```

Comment:
Can you add a `noarch: python` line here? ([docs](http://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages))